VideoLocker/src/main/java/org/edx/mobile/task/GetCommentListTask.java

```diff
@@ -10,17 +10,25 @@
 
     String threadId;
     int page = 1;
+    boolean isQuestionType;
+    boolean shouldGetEndorsed;
 
-    public GetCommentListTask(Context context, String threadId, int page) {
+    public GetCommentListTask(Context context, String threadId, int page, boolean isQuestionType,
+                              boolean shouldGetEndorsed) {
```

Comment:
`shouldGetEndorsed` & `getEndorsed` both convey the proper message, which `endorsed` doesn't. And since we're opting for the shorter one, so, lets go for `getEndorsed`.

That being said, imo boolean variables should always begin with helping verbs like is, has, should etc.